src/loaddb/load_session.cpp

```diff
@@ -199,10 +199,6 @@ namespace cubload
 	    m_session.stats_update_rows_committed (lines_inserted);
 	    m_session.stats_update_last_committed_line (line_no + 1);
 
-	    if (!m_session.get_args ().syntax_check && !m_session.get_args ().disable_statistics)
-	      {
-		m_session.append_log_msg (LOADDB_MSG_UPDATED_CLASS_STATS, class_name.c_str ());
```

Comment:
how does removing this log message impact loaddb output? Is it expected after each commit or not?

Reply:
SA_MODE does the update class stats after every class loaded :D so this was a bit incorrect.